query/templates/search/v710/topicCountQuery.tmpl

```diff
@@ -2,8 +2,19 @@
 "query" : {
     "bool" : {
       "must" : {
+                 {{- if .Term}}
+                     {{- template "coreQuery.tmpl" .}}
```

Comment:
Again can we be consistent with indentation to demonstrate nested objects

Reply:
Fixed in update 1.